sources/vi.cuutruyen/res/settings.json

```
@@ -1,9 +1,21 @@
 [
-  {
-    "type": "text",
-    "key": "proxy_url",
-    "placeholder": "HTTP Proxy URL",
-    "title": "Proxy URL",
-    "default": "https://hard-emu-60.deno.dev"
-  }
+	{
+		"type": "group",
+		"title": "Cài đặt",
```

you can use the app's localization.
```suggestion
		"title": "SETTINGS",
```